packages/text-input-react/package.json

```diff
@@ -37,6 +37,8 @@
         "dev": "run-p dev:*"
     },
     "dependencies": {
+        "@fremtind/jkl-icon-button": "^0.1.0",
```

Comment:
This is not needed. `jkl-icon-button-react` should handle its own style dependency to avoid conflicting versions.

Reply:
removed: c69343a8765bf8ab48659baf1ed963c441e7d332